src/decoder/mod.rs

```diff
@@ -54,15 +54,16 @@ impl OutputInfo {
 }
 
 #[derive(Clone, Copy, Debug)]
+/// Limits on the resources the `Decoder` is allowed too use
 pub struct Limits {
-    /// max number of pixels: `width * height` (default: 67M = 2<sup>26</sup>)
-    pub pixels: u64,
+    /// maximum number of bytes the decoder is allowed to allocate, default is 64M
```

Comment:
It might be helpful to clarify that this is Mib, not Mb. See https://en.wikipedia.org/wiki/Mebibyte

Reply:
Thanks, I always wondered which was which

Reply:
There's also 'the decoder will allocate less than 10K bytes' somewhere below, as `10*1024`.